src/core/grpc_service.proto

```diff
@@ -325,6 +325,12 @@ message SharedMemoryControlRequest
     //@@       To unregister all active shared memory regions.
     //@@
     UNREGISTER_ALL = 2;
+
```

Comment:
Having one request type for all the control requests doesn't work if the different requests need different inputs. Instead look at the TraceControl messages to see how you can use oneof to define different request (and response) for different request types.

Reply:
Done